### DISCUSSION TITLE
System.IO.FileNotFoundException: Could not load file or assembly 'Avalonia.Fonts.Inter, ...'.

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#10506] [General]
When testing on of the latest PR's I came across this issue. The solution is simple: Add

```<PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.999-cibuild0030865-beta" />```

to you project file. This may be true for the next ```11.0.0-preview6``` as well.

**Comment by ahopper** [comment]
Alternatively if you don't want to use the Inter font it can be replaced with this in App.xaml resources
```
	<FontFamily x:Key="ContentControlThemeFontFamily">avares://MyApp/Assets#MYFont</FontFamily>
```
and by not including the Inter PackageReference you avoid enlarging the app with an unused font.

I wonder if there should be a more explicit way to replace the font.
